### PR TITLE
feat: Brownie example and description

### PR DESCRIPTION
**Pull request by arianejasuwienas**
**Description**:
Brownie tool and analysis and usage example.

**Related issue(s)**:

#3071

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by jeromy-cannon — DISMISSED** [review]

File: tools/brownie-example/README.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+   In order to run the tests on the non-forked (no snapshot method available) network 
+   you need to have 2 accounts configured in brownie
+   
+   Bob's account can not be empty:
+   
+   ```shell
+   brownie accounts new bob
+   brownie accounts new alice
+   ```
+   Since the transfer of hbars is performed during the tests it is adviced to connect to the forked network instead of
```
Comment (ebadiere) [inline tools/brownie-example/README.md:208]: Spelling.  It should read "it is advised..."

**Review by ebadiere — COMMENTED** [review]
Spelling correction

File: tools/brownie-example/README.md (hunk 1 above)
Comment (arianejasuwienas) [reply tools/brownie-example/README.md]: Yes, you are right. Thank you for noticing it.

**Review by ebadiere — APPROVED** [review]
LG.

**Review by acuarica — DISMISSED** [review]
lg

**Review by nathanklick — APPROVED** [review]

**Review by jeromy-cannon — APPROVED** [review]

**Review by ebadiere — APPROVED** [review]

**Pull request merged**